pkg/api/vlabs/validate.go

```diff
@@ -371,7 +371,7 @@ func (a *Properties) validateMasterProfile() error {
 			return e
 		}
 		if !a.IsClusterAllVirtualMachineScaleSets() {
-			return errors.New("VirtualMachineScaleSets for master profile must be used together with virtualMachineScaleSets for agent profiles. Set \"availabilityProfile\" to \"VirtualMachineScaleSets\" for agent profiles")
+			return errors.New("VirtualMachineScaleSets for master profile must be used together with virtualMachineScaleSets for agent profiles. See \"availabilityProfile\" to \"VirtualMachineScaleSets\" for agent profiles")
 		}
```

Comment:
Is there a reason we don't want `Set \"availabilityProfile\" to \"VirtualMachineScaleSets\" for agent profiles` here?

Reply:
whoops. Major fail. Will correct .

Reply:
@ritazh Corrected. Thanks!